### DISCUSSION TITLE
Audio context within video plugin

### DISCUSSION THREAD

**Discussion by victoria0527** [jspsych/jsPsych#775] [Q&A] [answered]
I am trying to use the video-button-response from jspych 6.1.0 on google chrome (Version 83.0.4103.106). When I run the code, this error comes up:
jspsych.js:44 
The AudioContext was not allowed to start. It must be resumed (or created) after a user gesture on the page. https://goo.gl/7K7WLu

It appears that it was fixed by someone in the audio-button plugin (https://github.com/jspsych/jsPsych/issues/518), but I am not sure where to modify the audio context in the video plugin

**Comment by vijaymarupudi** [comment]
Is this the very first trial, or do you have something like html-button-response before this trial?

**Comment by victoria0527** [comment]
This is the first trial; I am trying to figure out how to work with videos in jspsych, so the code is very basic

```html
<!DOCTYPE html>
<html>
    <head>
        <title>My experiment</title>
        <script src="jspsych-6.1.0/jspsych.js"></script>
        <script src="jspsych-6.1.0/plugins/jspsych-html-keyboard-response.js"></script>
        <script src="jspsych-6.1.0/plugins/jspsych-video-button-response.js"></script>
        <link href="jspsych-6.1.0/css/jspsych.css" rel="stylesheet" type="text/css"></link>
    </head>
    <body></body>
    <script>

    var hello_trial = {
        type: 'html-keyboard-response',
        stimulus: 'Hello world!'
        }
    
    var trial = {
        type: 'video-button-response',
        sources: [
            'video/f_Ba_TK01.mpeg'
        ],
        choices: ['Happy','Sad','Angry','Peaceful'],
        prompt: '<p>Testing our video</p>'
    }

    jsPsych.init({
        timeline: [hello_trial, trial]
    })

    </script>
</html>
```

**Comment by vijaymarupudi** [comment, marked as answer]
That's the error then. You need a trial that requires user interaction (like a button response or instructions) before the video trial. I don't believe this is an issue with jsPsych.

**Comment by jodeleeuw** [comment]
Just to elaborate, the error message is harmless if things are working as expected.
